tests/languages/factor/comments_feature.test

```diff
@@ -0,0 +1,43 @@
+a! ! word
+!a ! word
+! comment
+!	bad
+! 	fine
+! "also a comment"
+! : ( -- ) ;
+! ! leading comment-like token
+   ! whitespace before
+words blah ! comment after code on a line
+
+----------------------------------------------------
+
+[
+	[ "conventionally-named-word", "a!"],
+	[ "comment", [ "! word" ] ],
+	[ "normal-word", "!a"],
+	[ "comment", [ "! word" ] ],
+	[ "comment", [ "! comment" ] ],
+
+	[ "normal-word", "!" ], [ "normal-word", "bad" ],
+	[ "comment", [ "! \tfine" ] ],
+
+	[ "comment", [ "! \"also a comment\"" ] ],
+	[ "comment", [ "! : ( -- ) ;" ] ],
+	[ "comment", [ "! ! leading comment-like token" ] ],
+	[ "comment", [ "! whitespace before" ] ],
+	[ "normal-word", "words" ],
+	[ "normal-word", "blah" ],
+	[ "comment", [ "! comment after code on a line" ] ]
+]
+
+----------------------------------------------------
+
+TODO: from the multiline vocab:

```

Comment:
What about this TODO?

Reply:
alright, i'll finish it later today

Reply:
the tests are not written for this yet, i'm sorry it takes so long on weekdays

Reply:
Don't worry about it. It's not like there's a deadline or anything.